mlptrain/potentials/_base.py

```diff
@@ -13,7 +14,7 @@ class MLPotential(ABC):
 
     def __init__(self,
                  name:   str,
-                 system: 'mlptrain.System'):
+                 system: 'mlt.System'):
```

Comment:
it would be really nice to make the type checking correct with something like [this](https://github.com/duartegroup/autodE/blob/4d6667592f083dfcf38de6b75c4222c0a0e7b60b/autode/species/species.py#L39-L40) – appreciate that's a bunch of work though. feel free to ignore 😄

Reply:
Heh, yes, this is on my TODO list, right now I did only a quick fix to make Ruff happy. Thanks for providing the example and for review! :)